storybookjs/storybook
# Sort the PropsTable rows alphabetically

Issue: #8539

## What I did

## How to test

- Is this testable with Jest or Chromatic screenshots? Yes Chromatic screenshots
- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.
